### ISSUE TITLE
sigmoid output?

### ISSUE THREAD

**Issue by godsmokescrack** [ternaus/TernausNet#17]
Your paper indicates a final sigmoid output layer, but the models in model.py do not have such a layer and the outputs are not in the range [0,1].  Do I just add a torch.sigmoid in the forward call?  In a somewhat unrelated matter, it doesn't seem that your loss function is available.  I'm new to machine learning/pytorch/computers and any help would be appreciated.  Thank you.
